builders/build/customize/adding-built-in-pallet.md

```diff
@@ -217,3 +217,35 @@ fn testnet_genesis(
 ```
 
 With the module included, this new runtime version has unlocked a new set of functionalities ready to be composed with even more of the Substrate built-in modules or custom-made ones.
+
+## Adding External Dependencies {: #adding-external-dependencies }
```

Comment:
The article is about adding a built in pallet, and we state in the introduction "This article focuses on the necessary steps for adding a built-in module to the EVM template."
This section seems to be a bit off here, we should move this content to the "Fixing Dependencies conflicts" article (which btw could be renamed to "Adding external dependencies"). wdyt?

Reply:
I was thinking that built-in are frame included pallets, and external dependencies would be those from third parties, such as an oracle